scripts/git-all.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+PS3='git command to run for all templates: '
+options=("pull" "push" "add" "commit" "checkout")
+
+select opt in "${options[@]}"
+do
+    case $opt in
+        "pull")
+            find ./templates -mindepth 1 -maxdepth 1 -type d -print -exec git -C {} pull \;
+            ;;
+        "push")
+            find ./templates -mindepth 1 -maxdepth 1 -type d -print -exec git -C {} push \;
+            ;;
+        "add")
+            echo What do you want to add?
```

Comment:
@Aaronkala looks good. Can you please quote all of these echos? As you can see its throwing the syntax highlighting off.
```suggestion
            echo "What do you want to add?"
```